torch/_dynamo/utils.py

```diff
@@ -1057,6 +1057,9 @@ def run_node(output_graph, node, args, kwargs, nnmodule):
             return nnmodule(*args, **kwargs)
         elif op == "get_attr":
             return output_graph.get_submodule(node.target)
+        elif op == "placeholder":
```

Comment:
j/w when does run_node get called on a placeholder?

Reply:
Over here: https://github.com/pytorch/pytorch/blob/6bf0e3b697ce688bc8325440dea3b51fea571c3d/torch/_dynamo/variables/tensor.py#L271. 
Basically when calling item on an input tensor.

Reply:
Lol, I mean, what is the case for it? how have we not needed it till now? The answer is "Operations invoked on inputs"